packages/select/src/select.vue

```diff
@@ -788,12 +793,8 @@
       },
 
       selectOption() {
-        if (!this.visible) {
```

Comment:
In our cognito fork, you could already cycle through before I got to this. The only reason i removed this condition is because I needed to disassociate selecting an option with toggling the menu and previously they were fairly entertwined.